src/client/pages/Offer/Introduction/Sidebar/Sidebar.tsx

```diff
@@ -52,21 +54,40 @@ const Wrapper = styled.div`
   }
 `
 
-const Container = styled.div`
-  width: ${SIDEBAR_WIDTH};
-  max-width: 100%;
-  padding: 1.25rem 1rem 1rem 1rem;
-  margin-top: 3rem;
-  background-color: ${colorsV3.white};
-  border-radius: 8px;
-  box-shadow: 0 8px 16px rgba(0, 0, 0, 0.08);
+type ContainerProps = {
+  stickyToTop?: boolean
+  stickyToBottom?: boolean
+}
 
-  ${LARGE_SCREEN_MEDIA_QUERY} {
-    position: fixed;
-    top: 8rem;
-    margin-top: 0;
-  }
-`
+const Container = styled.div(
+  ({ stickyToTop, stickyToBottom }: ContainerProps) => ({
+    width: SIDEBAR_WIDTH,
+    maxWidth: '100%',
+    padding: '1.25rem 1rem 1rem 1rem',
+    marginTop: '3rem',
+    backgroundColor: colorsV3.white,
+    borderRadius: '8px',
+    boxShadow: '0 8px 16px rgba(0, 0, 0, 0.08)',
+
+    [LARGE_SCREEN_MEDIA_QUERY]: stickyToTop
+      ? {
```

Comment:
This isn't very compact or beautiful, but does the job. Feel free to suggest an improvement 🙃

Reply:
And I still get to use multi-terneries in web-onboarding
![gif](https://media.giphy.com/media/3oKIPzLXQYb2Bn5PLG/giphy.gif)